docs/mine/spacerace.md

```diff
@@ -79,6 +79,10 @@ The [Competition Dashboard](https://spacerace.filecoin.io/) is where you can tra
 
 Block Rewards standings are listed on the [Block Rewards Dashboard](https://reward.testnet.filecoin.io/), and network stats on the [Testnet Network Stats](https://stats.testnet.filecoin.io/) page.
 
+#### I've registered my miner, however, I can't find it on the Space Race dashboard, anything else I need to do?
+
+Note that the [Space Race Dashboard](https://spacerace.filecoin.io) only shows deals result, which means you have to complete sealing at least one deal before your miner to be listed on the dashboard. 
```

Comment:
```suggestion
The [Space Race Dashboard](https://spacerace.filecoin.io) only shows miners after they have at least one sealed sector. To check if your miner has finished sealing its first sector, run `PLACEHOLDER FOR COMMAND` and look for `PLACEHOLDER FOR OUTPUT`.
```

Small clarification: there are 2 types of sectors that can be sealed: CommittedCapacity sectors (with no deals) and sectors with deals. The dashboard shows miners with at least one sector sealed (of either type). Typically in Space Race miners will start sealing a CC sector immediately, get on the power table/dash asap, and then start receiving deals from the dealbot to seal into additional sectors.

Please ask @arajasek or in Slack for the command & output to check for a first sealed sector.

Reply:
`lotus state active-sectors <minerId>` is a neat CLI way to do this (lists all active sectors).

Reply:
@mishmosh correct me if im wrong, according to [this](https://filecoinproject.slack.com/archives/G016VJ2J5DM/p1598601326014000?thread_ts=1598600903.012400&cid=G016VJ2J5DM) it seems like the deal must be sealed?

Reply:
> `lotus state active-sectors <minerId>` is a neat CLI way to do this (lists all active sectors).

and then they'll be using `lotus-miner sectors status <id>` to see the status of hat sector?

Reply:
@jennijuju Great question. That thread was referring to when *deal* success/fail would show up in the Storage Deal and Retrieval Deal columns - and yes that requires the deal to be sealed.

But this question (as phrased) is about when the miner itself is visible - and that is after the first sector is active.

Reply:
Ah I see thanks for confirming! Will make the change !